package.json

```diff
@@ -143,7 +143,7 @@
   "scripts": {
     "prepublishOnly": "echo \"Error: must publish from dist/\" && exit 1",
     "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest",
-    "test:integration": "yarn test --config=jest-integration-test.config.mjs",
+    "test:integration": "echo \"Disabled temporary\" && exit 1",
```

Comment:
does it work if you run in band?

Reply:
Can't run at all, we are `require()`ing, not `import()`, and `import()`+ runInBand not work too.

Reply:
Want try? Maybe you can find something.

Reply:
See #12271

Reply:
👍